src/main/resources/emissary/place/VersionPlace.cfg

```diff
@@ -0,0 +1,8 @@
+PLACE_NAME = "VersionPlace"
+SERVICE_NAME = "VERSION"
+SERVICE_TYPE = "VERIFY"
+SERVICE_DESCRIPTION = "add emissary version to payload"
+SERVICE_COST = 50
+SERVICE_QUALITY = 50
+
+SERVICE_PROXY = "UNKNOWN"
```

Comment:
Do we want to call this for everything, i.e. "*"?

Reply:
I didn't know if that was wanted, so for now just made it UNKNOWN

Reply:
Address in follow-on